js/core/utils/dom.js

```diff
@@ -130,6 +130,21 @@ var createMarkupFromString = function(str) {
     return tempElement.contents();
 };
 
+var extractTemplateMarkup = function(element) {
+    element = $(element);
+
+    var templateTag = element.length && element.filter(function isNotExecutableScript() {
+        var $node = $(this);
+        return $node.is("script[type]") && !(/\/(java|ecma)script/i.test($node.attr("type")));
```

Comment:
I'd use `indexOf("script")` for `type` checking.
Reference: https://www.w3.org/TR/html5/semantics-scripting.html#javascript-mime-type